### ISSUE TITLE
Server side error on server start that breaks addon (workshop version)

### ISSUE THREAD

**Issue by storm37000** [Grocel/3D-Stream-Radio#14]
An error prints to console on server start since the latest major workshop update.

```
[3D Stream Radio] lua/weapons/gmod_tool/stools/streamradio.lua:16: attempt to call field 'GetDefaultModel' (a nil value)
  1. unknown - lua/weapons/gmod_tool/stools/streamradio.lua:16
   2. include - [C]:-1
    3. unknown - gamemodes/sandbox/entities/weapons/gmod_tool/stool.lua:154
     4. include - [C]:-1
      5. unknown - gamemodes/sandbox/entities/weapons/gmod_tool/shared.lua:372
       6. include - [C]:-1
        7. unknown - gamemodes/sandbox/entities/weapons/gmod_tool/init.lua:10
```

**Comment by Grocel** [comment]
Thanks for reporting. This error was caused by the server being outdated.

I just released an update that adjust the version check to support the server version 230628 or newer.
Additionally the patch comes with a better error handling, which will reduce the likeliness of error spam.

If the addon does not load, it it will show you why when you try to use it. Spawned radio entities will be disabled, instead of spamming errors, etc.

The newest change also brings some optimizations and other bug fixes.
